### PR TITLE
Reduce the content in Gin-Mongo Smaple Doc 

### LINKED ISSUE #53: [ Documentation request ] Reduce the content in Gin-Mongo Smaple Doc
# Brief description

- When you visit [Gin-Mongo Sample](https://docs.keploy.io/docs/go/quickstart/gin-mongo-quickstart) page, the keploy server `installation methods` are listed again. We can add a redirection link instead of displaying the entire installation process. 

- We can add a [mongoDB](https://www.mongodb.com/) and [PostgreSQL](https://www.postgresql.org/) official website links like done for Gin and Echo for convenience.

Screenshot from 2022-09-30 22-20-53
Screenshot from 2022-09-30 22-23-23

- We can remove one of the line from documentation in [Gin-Mongo Sample](https://docs.keploy.io/docs/go/quickstart/gin-mongo-quickstart) 

Screenshot from 2022-09-30 22-24-59

#  Your recommended content

You can refer to the [`docs/go/quickstart`](https://github.com/keploy/docs/tree/main/docs/go/quickstart) folder.

### PR DESCRIPTION
**Pull request by Manoj0Marmat**
Signed-off-by: Manoj Marmat <manojrajput762000@gmail.com>

## Description

Fixes #53 

## Type of change

- [x] This change requires a documentation update.

## Additional Context (Please include any Screenshots/gifs if relevant)

Screenshot 2022-10-09 at 3 51 29 PM
Screenshot 2022-10-09 at 3 52 05 PM
Screenshot 2022-10-09 at 3 52 18 PM

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@SanskritiHarmukh  please review it

File: docs/go/quickstart/gin-mongo-quickstart.md [hunk 1]
```diff
@@ -13,17 +13,15 @@ tags:
...
 ### Pre-requisites
 
 - [Go](https://golang.org/doc/install)
 
-import ServerInstallation from '../../server/server-installation.md'
+## Installation Methods
 
-<ServerInstallation/>
-
-Once you have the Keploy Server running, open the Keploy Console at [http://localhost:8081](http://localhost:8081)
```
Comment (ShivamTyagi12345) [inline docs/go/quickstart/gin-mongo-quickstart.md:26]: I dont think deleting these is a good call

Comment (Manoj0Marmat) [reply docs/go/quickstart/gin-mongo-quickstart.md:26]: it's already mentioned in server installation 
Screenshot 2022-10-09 at 4 11 52 PM
and @SanskritiHarmukh mam mentioned
We can remove one of the line from documentation in [Gin-Mongo Sample](https://docs.keploy.io/docs/go/quickstart/gin-mongo-quickstart)
screen shot

Comment (ShivamTyagi12345) [reply docs/go/quickstart/gin-mongo-quickstart.md:26]: oh sure sure ! We can lead with this

**Comment by Manoj0Marmat** [comment]
@SanskritiHarmukh @nehagup please review it

File: docs/go/quickstart/gin-mongo-quickstart.md (hunk 1 above)
Comment (Manoj0Marmat) [reply docs/go/quickstart/gin-mongo-quickstart.md:26]: Please review it

**Comment by SanskritiHarmukh** [comment]
@Manoj0Marmat I'll review it and revert back to you. Don't worry.

**Comment by Manoj0Marmat** [comment]
@SanskritiHarmukh  k k 👍🏻

File: docs/go/quickstart/gin-mongo-quickstart.md [hunk 2]
```diff
@@ -13,17 +13,15 @@ tags:
...
+[Learn more](../../server/server-installation.md) about Installation Methods
```
Comment (SanskritiHarmukh) [inline docs/go/quickstart/gin-mongo-quickstart.md:24]: Reform this statement as- Navigate to [Installation guide](attach link here) to quickly install and run the keploy server.

File: docs/go/quickstart/gin-mongo-quickstart.md [hunk 3]
```diff
@@ -13,17 +13,15 @@ tags:
...
 
-In this sample application we're using 2 dependencies - [Gin](https://gin-gonic.com), mongoDB.
+In this sample application we're using 2 dependencies - [Gin](https://gin-gonic.com), [mongoDB](https://www.mongodb.com/).
 
 ### Pre-requisites
 
 - [Go](https://golang.org/doc/install)
 
-import ServerInstallation from '../../server/server-installation.md'
+## Installation Methods
```
Comment (SanskritiHarmukh) [inline docs/go/quickstart/gin-mongo-quickstart.md:22]: Change this heading to - Installation

File: docs/go/quickstart/gin-mongo-quickstart.md (hunk 2 above)
Comment (Manoj0Marmat) [reply docs/go/quickstart/gin-mongo-quickstart.md:24]: okay got it

**Comment by Manoj0Marmat** [comment]
@SanskritiHarmukh all mongo doc changes fixed please review it

**Pull request merged**